bedrock/base/templates/includes/news-feed.html

```diff
@@ -6,7 +6,7 @@
 {% if LANG.startswith('en') and articles %}
 <section class="news">
   <div class="content">
-    <h2 class="section-title"><span>{{ _('Fresh from our Blog') }}</span></h2>
+    <h2 class="section-title"><span>{{ _('Fresh From Our Blog') }}</span></h2>
```

Comment:
@peiying2 This is a string change but this component is currently en-US only and I don't know if this string has been exposed to any locales. If so, let me know. Since it's a shared component I'm not sure if a regular tag will work.